lib/fastlane/plugin/wpmreleasetoolkit/actions/get_prs_list_action.rb

```diff
@@ -0,0 +1,81 @@
+require 'fastlane/action'
+require_relative '../helper/ghhelper_helper'
+
+module Fastlane
+  module Actions
+    class GetPrsListAction < Action
+      def self.run(params)
+        repository = params[:repository]
+        start_tag = params[:start_tag]
+        end_tag = params[:end_tag]
+        report_path = params[:report_path]
+
+        # Get commit list
+        commit_list = sh("git log --pretty=oneline #{start_tag}..#{end_tag}")
+
+        # Extract PRs
+        pr_list = []
+        commit_list.split("\n").each do | commit |
+          if (commit.include?("Merge pull request #"))
+            # PR found, so extract PR number
+            pr_list.push(commit.partition('#').last.split(' ')[0])
+          end
+        end
+
+        # Get infos from GitHub and put into the target file
+        client = Fastlane::Helper::GhhelperHelper.GHClient()
+        File.open(report_path, "w") do | file |
+          pr_list.each do | pr_number |
+            data = client.pull_request(repository, pr_number.to_i)
+            file.puts("##{data[:number]}: #{data[:title]} @#{data[:user][:login]} #{data[:html_url]}")
+          end 
+        end 
+      end
+
+      def self.description
+        "Generate the list of the PRs from `start_tag` to `end_tag`"
+      end
+
+      def self.authors
+        ["Lorenzo Mattei"]
+      end
+
+      def self.return_value
+        # If your method provides a return value, you can describe here what it does
+      end
+
+      def self.details
+        # Optional:
+        "Generate the list of the PRs from `start_tag` to `end_tag`"
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :repository,
+                                   env_name: "GHHELPER_REPOSITORY",
+                                description: "The remote path of the GH repository on which we work",
+                                   optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :start_tag,
+                                description: "The tag from which the research starts",
```

Comment:
I'm maybe rename `research` to `report`? (and for the end tag too)

Reply:
Thank you @frosty! Descriptions updated!